org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -212,7 +212,7 @@ GithubOidcSageBionetworksSvcCatalogLibrary:
   Type: update-stacks
   DependsOn: GithubOidcSageBionetworks
   Template: github-oidc-provider-access.njk
-  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-svc-catalog-library
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-service-catalog-library
```

Comment:
if you are changing `svc` to `service` then please change it in all of the other locations.